iam/api-client/src/main/java/iam/snippets/Quickstart.java

```diff
@@ -13,48 +13,161 @@
     GoogleCredentials credential =
         GoogleCredentials.getApplicationDefault()
             .createScoped(Collections.singleton(IamScopes.CLOUD_PLATFORM));
 
-    // Create the Cloud IAM service object
-    Iam service =
-        new Iam.Builder(
+    // Creates the Cloud Resource Manager service object.
+    CloudResourceManager service =
+        new CloudResourceManager.Builder(
                 GoogleNetHttpTransport.newTrustedTransport(),
                 JacksonFactory.getDefaultInstance(),
                 new HttpCredentialsAdapter(credential))
-            .setApplicationName("quickstart")
+            .setApplicationName("service-accounts")
             .build();
+    return service;
+  }
+
+  public static void addBinding(
+      CloudResourceManager crmService, String projectId, String member, String role) {
+
+    // Gets the project's policy.
+    Policy policy = getPolicy(crmService, projectId);
 
-    // Call the Cloud IAM Roles API
-    ListRolesResponse response = service.roles().list().execute();
-    List<Role> roles = response.getRoles();
+    // Finds binding in policy, if it exists
+    Binding binding = null;
+    for (Binding b : policy.getBindings()) {
+      if (b.getRole().equals(role)) {
+        binding = b; 
+        break;
+      }
+    }
+
+    if (binding != null) {
+      // If binding already exists, adds member to binding.
+      binding.getMembers().add(member);
+    } else {
+      // If binding does not exist, adds binding to policy.
+      binding = new Binding();
+      binding.setRole(role);
+      binding.setMembers(Collections.singletonList(member));
+      policy.getBindings().add(binding);
+    }
+
+    // Sets the updated policy
+    setPolicy(crmService, projectId, policy);
+  }
+
+  public static void removeMember(
+      CloudResourceManager crmService, String projectId, String member, String role) {
+    // Gets the project's policy.
+    Policy policy = getPolicy(crmService, projectId);
+
+    // Removes the member from the role.
+    Binding binding = null;
+    for (Binding b : policy.getBindings()) {
+      if (b.getRole().equals(role)) {
+        binding = b;
+        break;
+      }
+    }
+    if (binding.getMembers().contains(member)) {
+      binding.getMembers().remove(member);
+      if (binding.getMembers().isEmpty()) {
+        policy.getBindings().remove(binding);
+      }
+    }
+
+    // Sets the updated policy.
+    setPolicy(crmService, projectId, policy);
+  }
+
+  public static Policy getPolicy(CloudResourceManager crmService, String projectId) {
+    // Gets the project's policy by calling the
+    // Cloud Resource Manager Projects API.
+    Policy policy = null;
+    try {
+      GetIamPolicyRequest request = new GetIamPolicyRequest();
+      policy = crmService.projects().getIamPolicy(projectId, request).execute();
+    } catch (IOException e) {
+      System.out.println("Unable to get policy: \n" + e.toString());
+    }
```

Comment:
Better to just add a throws to getPolicy and not catch `IOException` if you aren't doing anything with it.